CHANGELOG.md

```diff
@@ -6,20 +6,26 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.10.0 (2023-02-16)
+
 **Features**
+* #4874 Allow joins in continuous aggregates
 * #5241 Allow RETURNING clause when inserting into compressed chunks
-* #5245 Mange life-cycle of connections via memory contexts
+* #5243 Enable real-time aggregation for continuous aggregates with joins
+* #5245 Manage life-cycle of connections via memory contexts
 * #5246 Make connection establishment interruptible
 * #5253 Make data node command execution interruptible
-* #5243 Enable real-time aggregation for continuous aggregates with joins
 * #5262 Extend enabling compression on a continuous aggregrate with 'compress_segmentby' and 'compress_orderby' parameters
 
 **Bugfixes**
 * #4926 Fix corruption when inserting into compressed chunks
```

Comment:
@mahipv this is one of the major features for 2.10.0, not a bug fix. Let's move it up in the features section

Also the title is not correct, it should be: `Refactor INSERT into compressed chunks`